db/view/view.cc

```diff
@@ -1725,7 +1725,7 @@ bool should_generate_view_updates_on_this_shard(const schema_ptr& base, const lo
 //
 // If the assumption that the given base token belongs to this replica
 // does not hold, we return an empty optional.
```

Comment:
This comment got outdated, now we'd be returning a pair.

Reply:
I added a more extensive description in the rebase